server/src/internalClusterTest/java/org/opensearch/snapshots/SearchableSnapshotIT.java

```diff
@@ -104,6 +109,76 @@ public void testCreateSearchableSnapshot() throws Exception {
         assertIndexDirectoryDoesNotExist("test-idx-1-copy", "test-idx-2-copy");
     }
 
+    public void testSearchableSnapshotIndexIsReadOnly() throws Exception {
+        final String indexName = "test-index";
+        final Client client = client();
+        createRepository("test-repo", "fs");
+        createIndex(
+            indexName,
+            Settings.builder()
+                .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, "0")
+                .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, "1")
+                .build()
+        );
+        ensureGreen();
+        indexRandomDocs("test-idx-1", 100);
```

Comment:
Should use `indexName` here. Or remove this line. It's probably not necessary to index any documents.